packages/sdk-react/src/components/auth/Auth.tsx

```diff
@@ -186,48 +192,25 @@ const Auth: React.FC<AuthProps> = ({
         }
       }
 
-      const sessionResponse = await passkeyClient?.createReadWriteSession({
-        targetPublicKey: authIframeClient?.iframePublicKey!,
-        ...(authConfig.sessionLengthSeconds !== undefined && {
-          expirationSeconds: authConfig.sessionLengthSeconds.toString(),
-        }),
-        organizationId:
-          turnkey?.config.defaultOrganizationId ??
-          process.env.NEXT_PUBLIC_ORGANIZATION_ID!,
-      });
-      if (sessionResponse?.credentialBundle) {
-        await handleAuthSuccess(
-          sessionResponse.credentialBundle,
-          authConfig.sessionLengthSeconds?.toString(),
-        );
-      } else {
-        setPasskeySignupError(authErrors.passkey.loginFailed);
-      }
+      await passkeyClient?.loginWithPasskey(
+        SessionType.READ_WRITE,
```

Comment:
did we want to be able to allow this to be overridden? given

```
loginWithPasskey - attempts to authenticate a user via passkey and create a read-only or read-write session
```

or perhaps not in this case, given this is our pre-made/built-in modal

Reply:
in any case, so much cleaner!!